admin-console/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
+- The critical alerts group should be expanded by default  and all others should be collapsed.
+
+## Available Updates
+![](img/notifications-updates.png)
+- Updates for the cluster or for specific operators could also be listed here.
+- Clicking on one of these cards would take the user to a page with information about the updates.
+- Action buttons to install the update immediately could also be provided.
+
+## Empty State
+![](img/notifications-empty.png)
+- When there are no critical alerts firing, the `Critical alerts` group should still be present but should contain an empty state pattern.
+- The `Available updates` category and any additional categories should be hidden when they are empty.
+
+## First Time Setup
```

Comment:
I think this is a great feature, but how about some sort of empty state treatment rather than a firing alert treatment?

Reply:
Can you elaborate?

Reply:
I was thinking something more like a small version of when receivers need to be created in general: http://openshift.github.io/openshift-origin-design/admin-perspective/monitoring/alertmanager-config/img/config-empty.png